### PR TITLE
update EDL_description.md

### PR DESCRIPTION
**Pull request by adi88d**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [link to the issue](https://jira-dc.paloaltonetworks.com/browse/CIAC-6795)

## Description
- Added the *Listen Port* and *credentials* parameters to XSOAR SAAS.
- Update the the relevant sections for **XSIAM**, **XSOAR_SAAS** and **XSOAR_ON_PREM** in `EDL_description.md` 

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by adi88d** [comment]
### XSIAM section:

image

### XSOAR SAAS section:
image
image

File: Packs/EDL/ReleaseNotes/3_2_16.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### Generic Export Indicators Service
+- Added the *Listen Port* and *credentials* parameters to XSOAR SAAS.
```
Comment (DeanArbel) [inline Packs/EDL/ReleaseNotes/3_2_16.md:5]:
```suggestion
- Added back the *Listen Port* and *Username* parameters for XSOAR SaaS.
```

**Review by DeanArbel — APPROVED** [review]
See my comment about the release notes.

File: Packs/EDL/Integrations/EDL/EDL_description.md [hunk 2]
```diff
@@ -27,44 +18,50 @@ In a web browser, go to **http://<cortex-xsoar-server-address>:<listen_port>**.
...
+
 **Note**: Do not set `username` and `password` in the integration instance if you are running the integration via the hosted instance. The `username` and `password` fields are for usage when running the integration via an on-prem engine.
 
 **Note**: If no `Listen Port` param was given and the test button was clicked, the test will run with the default port 1111. After pressing `save & exit` a new free port will be assigned to the `Listen Port` parameter automatically.
 
-1. To access the **Generic Export Indicators Service** by instance name, set up the **username** and **password** values in the **External Dynamic List Integration** page (**Settings** > **Configurations** > **Integrations** > **External Dynamic List Integration**).
+**Note**: The External Dynamic List is not accessible via web browsers and you will receive an unauthorized error if accessing the External Dynamic List via a browser.
+
+**Note**: When using instance that running on engine, you can access only with the `Listen Port`, go to **http://<cortex-xsoar-server-address>:<listen_port>**.
```
Comment (ShirleyDenkberg) [inline Packs/EDL/Integrations/EDL/EDL_description.md:35]:
```suggestion
**Note**: When using an instance that is running on an engine, you can access only with the `Listen Port`. Go to **http://<cortex-xsoar-server-address>:<listen_port>**.
```

**Comment by ShirleyDenkberg** [comment]
@DeanArbel @omerKarkKatz Doc review completed.

**Pull request merged**